### PR TITLE
NSString+URBN - added convenience category on NSString

### PR DESCRIPTION
**Pull request by rmigneco**
- Added a category with two functions
- `containsString` which determines if a substring is present with case sensitivity 
- `containsCaseInsensitiveString` which determines if a substring is present without case sensitivity

### CODE REVIEW

File: Pod/Classes/NSString+URBN.m [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+//
+//
+
+#import "NSString+URBN.h"
+#import "URBNMacros.h"
+
+@implementation NSString (URBN)
+
+- (BOOL)urbn_containsString:(NSString *)string {
+    return ([self urbn_containsString:string withCaseOption:0]);
```
Comment (jgrandelli) [inline Pod/Classes/NSString+URBN.m]: Please use enum name for `withCaseOption`.

**Comment by jgrandelli** [comment]
Pretty good. The comment about the enum plus you should add some unit tests for this. Let me know if you need help with those.

File: Pod/Classes/NSString+URBN.m (hunk 1 above)
Comment (rmigneco) [inline Pod/Classes/NSString+URBN.m]: Do you mean replace `withCaseOption` with `withNSStringCompareOption`?

**Comment by jgrandelli** [comment]
make sure to push this to trunk.

**Pull request merged**